Marlin/configuration_store.cpp

```diff
@@ -1258,7 +1332,51 @@ void MarlinSettings::postprocess() {
         }
       #else
         uint16_t val;
-        for (uint8_t q=11; q--;) EEPROM_READ(val);
+        for (uint8_t q=TMC_AXES; q--;) EEPROM_READ(val);
+      #endif
+
+      #if HAS_TRINAMIC
+        #define TMC_SET_PWMTHRS(N,P,Q) tmc_set_pwmthrs(stepper##Q, TMC_##Q, tmc_hybrid_threshold[P##_AXIS], planner.axis_steps_per_mm[P##_AXIS])
```

Comment:
No `N` argument needed?

Reply:
Already patched. It is needed as an index for the array.